ocs_ci/ocs/scale_lib.py

```diff
@@ -168,7 +173,8 @@ def create_multi_pvc_pod(
         return pod_objs, pvc_objs
 
     def create_scale_pods(
-        self, scale_count=1500, pods_per_iter=5, io_runtime=None, start_io=None
+        self, scale_count=1500, pods_per_iter=5, io_runtime=None,
+        pvc_size=None, start_io=None
```

Comment:
Please make sure the signature change is not affecting existing usages.

Reply:
It will not change the existing usages.  Added doc string for create_scale_pods() as well.